incubator/azuremonitor-containers/templates/omsagent-deployment.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if and (ne .Values.omsagent.secret.key "<your_workspace_key>") (ne .Values.omsagent.secret.wsid "<your_workspace_id>") (ne .Values.omsagent.env.clusterName "<your_cluster_name>")}}
-apiVersion: apps/v1
+apiVersion: extensions/v1beta1
```

Comment:
Why are we changing this back to a non-stable API version?

Reply:
Update PR comments. We will update the API version once subsequent PR along with changes to add the support for 1.16 version.  reverting API version to extensions/v1beta1 to get the chart working since this change API version change broke the chart.